### ISSUE TITLE
Networking programs

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#7]
Build FTP, TELNET, SUPDUP.  More?

**Comment by larsbrinkhoff** [comment]
In my opinion, I think it's nice if this is split into many commits.  One commit for each program.  The commit should include source code, documentation, build script, and README updates.

**Comment by eswenson1** [comment]
ejs/net2 branch now has FTP, TELNET, and SUPDUP. I think open-ended tickets lead to confusion, so I'd like to close this ticket once ejs/net2 is merged to master, and open specific tickets for other missing programs (e.g. NAME, WHOIS, HOSTS, MLDEV, MLSLV).  And probably another ticket for CHAOS net support.

**Comment by larsbrinkhoff** [comment]
Good idea.  Closing this.

**Comment by larsbrinkhoff** [comment]
Sorry, it was a bit premature.
